DataFormats/Math/interface/CMSUnits.h

```diff
@@ -0,0 +1,36 @@
+#ifndef DataFormats_Math_CMS_Units_h
+#define DataFormats_Math_CMS_Units_h
+
+// This file provides units represented with user-defined literals to more easily attach units to numerical values.
+// The CMS convention is that centimeter = 1 and GeV = 1
+
+#include "DataFormats/Math/interface/deltaPhi.h"
```

Comment:
why is this needed?
It looks like I was not paying enough attention to the review of #26227
I think that these angle_units and angle0to2pi should be just moved in here or to their separate file. FWIW, they have nothing to do with the deltaPhi method implementations.

Reply:
angle0to2pi does belong in `deltaPhi.h`. That file contains `reduceRange`, which reduces an angle to the range -pi to pi. angle0to2pi is merely a variation of `reduceRange` that reduces an angle to the 0 to 2pi range, so they do go together.

angle_units was put into `deltaPhi.h` because they are all related to angles, and I didn't want to create another file if it wasn't necessary. Any new header file might cause confusion with the vaguely named `angle.h` in the same directory. But if you want to move angle_units to its own header file, that would be OK with me. Or maybe angle_units should go into `angle.h`?

Reply:
angle_units.h sounds quite OK.

For the angle0to2pi I added a note in https://github.com/cms-sw/cmssw/pull/26227/files#r309469496

Reply:
@slava77 - from a user point of view it's better to have one CMSUnits header file to handle all CMS units including their conversions such as ```convertRadToDeg``` rather then CMSLengthUnits, CMSAngleUnits, ...

Reply:
aren't the angles common in GEANT and CMS context?
The length units are not the same.

Reply:
@slava77 - yes, they are the same.